.circleci/config.yml

```diff
@@ -90,6 +90,9 @@ references:
 jobs:
   run-tests:
     working_directory: ~/wp-calypso
+    environment:
+      DETECT_CHROMEDRIVER_VERSION: 'false'
```

Comment:
See https://www.npmjs.com/package/chromedriver#detect-chromedriver-version.
The doc doesn't say it, but by default this feature is enabled. I had to disable it in order to make it use `CHROMEDRIVER_FILEPATH`